### DISCUSSION TITLE
setTableDataClass not working

### DISCUSSION THREAD

**Discussion by Restingo** [rappasoft/laravel-livewire-tables#321] [Q&A] [answered]
Like in #94 and #95 I tried to style td with function `setTableDataClass` but it's not working (anymore). What can I do?

**Comment by rappasoft** [comment, marked as answer]
We for some reason got rid of that functionality at 1.0, but you can add classes to column headers (https://github.com/rappasoft/laravel-livewire-tables/wiki/Misc.-Column-Functionality)

Maybe I'll add it back in the next release if I can figure out why I removed it in the first place.

**Reply by rappasoft** [reply]
Next release: https://github.com/rappasoft/laravel-livewire-tables/commit/3425ebd3c4926be448e1de962a0f660e9017f590

Today or tomorrow.
